src/commands/synthetics/utils/internal.ts

```diff
@@ -90,7 +90,7 @@ export const toExecutionRule = (env: string | undefined): ExecutionRule | undefi
   return undefined
 }
 
-export const toStringObject = (env: string | undefined): {[key: string]: string} | undefined => {
+export const toStringObject = (env: string | undefined): StringObject | undefined => {
```

Comment:
I'd call this a `StringMap` 😁

Anything in JS is an object, so "string object" is a pleonasm

Reply:
TIL - the meaning of pleonasm :D